### DISCUSSION TITLE
IPTV Playlist is not loading

### DISCUSSION THREAD

**Discussion by tbreitha** [AndreyPavlenko/Fermata#553] [General]
Hello Andrey,
sinc a few weeks my paid IPTV playlist is not loading anymore, but only in fermata. Emby or my other IPTV Player like IPTV Pro 8.21 on Android 15 is working perfectly. Only Fermata is showing 0 channels and 0 groups. It would be really great, if you can have a look at it. I can send you a test-line for the iptv provider to verify the issue and hopefully can fix it...... Please let me know
Cheers Tom

**Comment by AndreyPavlenko** [comment]
Have you tried to change the user agent in the playlist settings?
If you can share the link, please send me by email. I'll check.

**Reply by tbreitha** [reply]
what options can i use as user agent

**Reply by AndreyPavlenko** [reply]
Try Google

**Reply by tbreitha** [reply]
Whats your E-mail address?

**Reply by AndreyPavlenko** [reply]
andrey.a.pavlenko@gmail.com

**Reply by tbreitha** [reply]
Not sure of the syntax, at the moment it is set to: Dalvik/2.1.0 (Linux; U; Android 15; Pixel 9 Build/AP3A.241105.008)

**Reply by tbreitha** [reply]
sent you an e-mail

**Reply by AndreyPavlenko** [reply]
If you can download the playlist in the browser, open this https://www.whatsmyua.info/ link in the same browser and copy the string.

**Reply by tbreitha** [reply]
hm still not working. i sent you the playlist url, can you check on your site, please?

**Reply by tbreitha** [reply]
Tried different things, but it still don't work.

**Comment by lucassssmi** [comment]
Hello to every body .
I have the same problem, with thi playlist
But before it worked .. 

http://yaa.com/get.php?username=a******B&password=*****N&type=m3u_plus&output=ts

Pealse, Do you can help 


<img width="870" height="703" alt="fermata" src="https://github.com/user-attachments/assets/028ac466-5395-4c60-b3a7-8e4127e36c1d" />
